<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formRequest</class>
 <widget class="QDialog" name="formRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YoutubeAudioDownloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resources/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Download audios from youtube channels!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtChannel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="placeholderText">
    <string> Insert the youtube channel, e. g. nprmusic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 19, 18);
color: rgb(255, 255, 255);
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For example:&lt;/p&gt;&lt;p&gt;If the url channel is like this: https://www.youtube.com/c/&lt;span style=&quot; font-weight:700;&quot;&gt;nprmusic&lt;/span&gt;&lt;br/&gt;or https://www.youtube.com/@&lt;span style=&quot; font-weight:700;&quot;&gt;kexp&lt;/span&gt;, insert &lt;span style=&quot; font-weight:700;&quot;&gt;nprmusic &lt;/span&gt;or &lt;span style=&quot; font-weight:700;&quot;&gt;kexp &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
